scripts/setup-initial-patch.sh

```diff
@@ -48,6 +48,17 @@ sed -i -e "s/boring.VerifyECDSA(b, hash, r, s)/boring.VerifyECDSA(b, hash, r, s,
 sed -i -e "s/boring\.Enabled/boring\.Enabled()/g" ${GO_SOURCES}
 sed -i -e "s/\"crypto\/internal\/boring\"/boring \"crypto\/internal\/backend\"/g" ${GO_SOURCES}
 sed -i -e "s/const boringEnabled/var boringEnabled/g" ${GO_SOURCES}
+sed -i -e "s/testConfig.Clone()/testConfigTemplate()/g" src/crypto/tls/boring_test.go
```

Comment:
nit: `.` in the pattern should be escaped as in line 48

Reply:
Done, thanks!